### DISCUSSION TITLE
Give Spectators Something to do.

### DISCUSSION THREAD

**Discussion by JamesTCrenshaw** [FakeFishGames/Barotrauma#4813] [Ideas and suggestions]
I am unsure of what exactly could be implemented, but even having some simple puzzle or something to that effect would satisfy the need. Spectators can have nothing to do for very long periods of time (upwards of 30 minutes), and even small puzzles or other interactable things would offer a significant respite from mere observation.

**Comment by RadCatish** [comment]
Perhaps a way to get your skills up a bit before you respawn?

**Comment by mLuby** [comment]
Let them take control of bots (or if there are none, nearby monsters 😈)!

**Reply by RadCatish** [reply]
I feel like people would start suiciding 🤣

**Comment by Sviat** [comment]
What about allowing them assigning commands to bots? Should be accepted by Captain/.
